git-infer

```diff
@@ -15,23 +21,24 @@ BEGIN {
   updatedCount = 0;
   addedCount = 0;
   deletedCount = 0;
+  newCount = 0;
 }
-
 $1 == "M" {
   updated[updatedCount] = $2;
   updatedCount += 1
 }
-
 $1 == "A" {
   added[addedCount] = $2;
   addedCount += 1
 }
-
 $1 == "D" {
   deleted[deletedCount] = $2;
   deletedCount += 1
 }
-
+$1 == "??" {
```

Comment:
Does it make sense to distinguish added files that are staged and untracked files? Wouldn't it be simpler to just add `|| $1 == "??"` to the "added" block?